osdk/README.md

```diff
@@ -0,0 +1,163 @@
+# Accelerate OS development with Asterinas OSDK
+
+[![Crates.io](https://img.shields.io/crates/v/cargo-osdk.svg)](https://crates.io/crates/cargo-osdk)
+
+### What is it?
+
+OSDK (short for Operating System Development Kit) is designed to simplify the development of Rust operating systems. It aims to streamline the process by leveraging the framekernel architecture, originally proposed by [Asterinas](https://github.com/asterinas/asterinas).
+
+`cargo-osdk` is a command-line tool that facilitates project management for those developed on the framekernel architecture. Much like Cargo for Rust projects, `cargo-osdk` enables building, running, and testing projects conveniently.
+
+### Install the tool
+
+#### Requirements
+
+Currenly, `cargo-osdk` only supports x86_64 ubuntu system. 
+
+`cargo-osdk` requires the following tools to be installed: 
+- Rust >= 1.75.0
+- Gcc compiler
+
+About how to install Rust, you can refer to the [official site](https://www.rust-lang.org/tools/install).
+
+Gcc compiler can be installed by
+```bash
+apt install build-essential
+```
+
+#### Install 
+
+Then, `cargo-osdk` can be installed by
+```bash
+cargo install cargo-osdk
+``` 
+
+#### Upgrade
+
+If `cargo-osdk` is already installed, the tool can be upgraded by
+```bash
+cargo install --force cargo-osdk
+```
+
+### Get start
+
+Here we provide a simple demo to demonstrate how to create and run a simple kernel with `cargo-osdk`.
+
+Suppose you are on a x86_64 ubuntu machine, to run a kernel with QEMU, the following tools should be installed:
+- qemu-system-x86_64
+- grub-mkrescue
+- ovmf 
+- xorriso
```

Comment:
I think the instructions to installing these tools should be moved to the Requirement section.